doc/version_1/062_relatedSystems.md

```diff
@@ -0,0 +1,107 @@
+
+## Related Vulnerability Management Systems
+
+There are several other bodies of work that are used in practice to assist vulnerability managers in making decisions.
+Three relevant systems are CVSS [@cvss_v3-1], EPSS [@jacobs2019exploit], and Tenable's Vulnerability Priority Rating ([VPR](https://www.tenable.com/blog/what-is-vpr-and-how-is-it-different-from-cvss)).
+There are other systems derived from CVSS, such as RVSS for robots and MITRE's effort to adapt CVSS to medical devices.
```

Comment:
Can we include a reference at least and possibly a bit of text for the medical device CVSS rubric in the CVSS-derived section? Maybe just references for the rubric and RSS?

Reply:
https://www.mitre.org/publications/technical-papers/rubric-for-applying-cvss-to-medical-devices

Reply:
or https://www.mitre.org/sites/default/files/publications/pr-18-2208-rubric-for-applying-cvss-to-medical-devices.pdf

Reply:
https://arxiv.org/abs/1807.10357